app/components/govuk_component/header.html.erb

```diff
@@ -30,11 +30,9 @@
       <% end %>
 
       <% if items.any? %>
-        <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide navigation menu">
-          <%= menu_button_label %>
-        </button>
+        <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="<%= menu_button_label %>">Menu</button>
```

Comment:
I think I must've misunderstood your previous request. I thought `menu_button_label` would control the content of the button on mobile:

![Screenshot from 2021-05-04 18-00-49](https://user-images.githubusercontent.com/128088/117041518-e2b53b80-ad02-11eb-940f-4fc8e7095aff.png)

Should it be renamed to `menu_button_aria_label` perhaps?

Reply:
> Should it be renamed to menu_button_aria_label perhaps?

I don’t think so. I really do believe that, where possible, this gem should share the same names and values as those used by the design system Nunjucks components. Unfortunate that in this case this is a breaking change, but maybe still early enough to mitigate?